### PR TITLE
Round

### PR DESCRIPTION
**Pull request by caub**
attempt for #694, it's not urgent, I'm not 100% sure it's good

last commit, I had to change a little test data, 
```
 -    <path d="M60 0a5 5 0 1 0 .001 10.001A5 5 0 0 0 60 0z"/>
 +    <path d="M60 0a5 5 0 1 0 .001 10.001 5 5 0 0 0 0-10.001z"/>
```
the 2 svgs look the same though https://jsfiddle.net/crl/4gzwfyvg/

### CODE REVIEW

File: plugins/convertPathData.js [hunk 1]
```diff
@@ -408,34 +400,34 @@ function filters(path, params) {
             // Rounding relative coordinates, taking in account accummulating error
             // to get closer to absolute coordinates. Sum of rounded value remains same:
             // l .25 3 .25 2 .25 3 .25 2 -> l .3 3 .2 2 .3 3 .2 2
-            if (precision !== false) {
```
Comment (caub) [inline plugins/convertPathData.js]: removed that check, I don't think that's useful, precision=false would keep raw values?

Comment (GreLI) [inline plugins/convertPathData.js]: Yep, that's right/

File: test/plugins/convertPathData.14.svg [hunk 2]
```diff
@@ -17,7 +17,7 @@
     <path d="M15 10a5 5 0 1 1 5-5l-5 5"/>
     <path d="M41.008.102a5.006 5.006 0 0 1 3.849 3.705"/>
     <path d="M7.234 19.474a5 5 0 0 1-4.933-8.683"/>
-    <path d="M60 0a5 5 0 1 0 .001 10.001A5 5 0 0 0 60 0z"/>
+    <path d="M60 0a5 5 0 1 0 .001 10.001 5 5 0 0 0 0-10.001z"/>
```
Comment (GreLI) [inline test/plugins/convertPathData.14.svg]: 4 characters longer, it's not good. Here is some problem.

Comment (caub) [inline test/plugins/convertPathData.14.svg]: ok right, but I think it's random, I'll try to find a case where it's the opposite, round shorter and toFixed longer

**Comment by caub** [comment]
I understand the fear of floating points error with Math.round(x * 1000) / 1000, but it can't happen

.999 factor in smartRound is a bit hacky, but it's just to pass the very rare case that round badly like `Math.round(9.995 * 100)/100 /* 9.99 instead of 1*/`  because 9.995*100 is 999.499999999

**Comment by caub** [comment]
Closing, and just commit the removed globals in cleanup PR

**Pull request closed without merging**